CampaignConfig/mdc2020_trkana.cfg

```diff
@@ -0,0 +1,33 @@
+[global]
+includes = /mu2e/app/users/sophie/ProductionManager/Production/CampaignConfig/mdc2020_prolog.cfg
+release = MDC2020
+desc = MDC2020s
+release_v_i = r
+release_v_o = s
+db_version = v1_0
+db_purpose = perfect
+primary_name = IPAMuminusFlateMinus
+trkana_version_counter = a
+trkana_version = v01_01_00
+beam = 1BB
```

Comment:
Is this parameter needed for TrkAna?  I would think not.

Reply:
No, I can remove it

Reply:
Hmm, I'm wondering. You might want to keep this, right, if you were doing studies.